bundles/org.openhab.binding.lifx/src/main/resources/ESH-INF/thing/tile.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="lifx"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="tile">
+		<label>LIFX Tile</label>
+		<channels>
+			<channel id="color" typeId="color" />
+			<channel id="temperature" typeId="temperature" />
+			<channel id="signalstrength" typeId="system.signal-strength" />
+			<channel id="tileeffecttype" typeId="tileeffecttype" />
+			<channel id="tilemorphspeed" typeId="tilemorphspeed" />
+			<channel id="tileflamespeed" typeId="tileflamespeed" />
```

Comment:
This should be a channel configuration parameter and not a channel itself (see other comment).

```suggestion
```